modules/gapi/src/backends/common/gbackend.hpp

```diff
@@ -58,6 +58,35 @@ namespace gimpl {
     struct Data;
     struct RcDesc;
 
+    struct RMatMediaBGRAdapter final: public cv::RMat::Adapter
```

Comment:
I'm not sure that `gbackend.hpp` is a good place to put the adapter

Reply:
Why not ?

Reply:
`RMatMediaBGRAdapter` is rather specific to put it in generic `gbackend`. If you don't have more appropriate location let's leave it here but at least put a comment suggesting to move it to another location

Reply:
It's needed to allow every backend use that

Reply:
It isn't specific for `streaming`, is it ?

Reply:
Ok, got you